github_pr.py

```diff
@@ -21,11 +22,12 @@ class OwnerCannotShipError(Exception):
 class NoMergeCommentError(Exception):
     pass
 
+logger = logging.getLogger()
 
 def check_required_fields(required, **args):
     for i in required:
         if args[i] is None:
-            print 'Error: %s parameter not set' % i
+            logging.error(' %s parameter not set', i.upper())
```

Comment:
i changed this, but im now seeing that none of my logging is working
my guess is because logger isnt defined in the other methods.